File: Array/easy/pb_1752/code.py
class Solution(object):
    def check(self, nums):
        """
        :type nums: List[int]
        :rtype: bool
        """
        n=len(nums)
        if n==0 or n==1:
            return True
        flag=0
        for i in range(n):
            if nums[i]>nums[(i+1)%n]:
                flag+=1
        return flag<=1
        

File: Array/easy/pb_1752/code.cpp
class Solution {
public:
    bool check(vector<int>& nums) {
        int len=nums.size();
        if(len==0||len==1) return true;
        int flag = 0;
        for(int i=0;i<len;i++){
            if(nums[i] > nums[(i+1)%len]) flag++;
            if(flag>1) return false;
        }
        return true;
    }
};